test_runner/src/main/kotlin/ftl/adapter/google/LocalesAdapter.kt

```diff
@@ -0,0 +1,66 @@
+package ftl.adapter.google
+
+import ftl.api.Locale
+import ftl.environment.TAGS
+import ftl.environment.TestEnvironmentInfo
+import ftl.environment.createTableColumnFor
+import ftl.environment.getOrCreateList
+import ftl.environment.tagToSystemOutColorMapper
+import ftl.run.exception.FlankGeneralError
+import ftl.util.applyColorsUsing
+import ftl.util.buildTable
+
+internal fun List<com.google.testing.model.Locale>.toApiModel(): List<Locale> = map { locale ->
```

Comment:
please use named import (ex. `GoogleApiLocale`) instead of the full package name

Reply:
Cannot be done. There are two names. One of them needs to be the full package name. We are migrating from one Locale to the other.

Reply:
IMO it's totally ok to use the full package name when conflict with names is occurring.

Reply:
IMO it's totally ok to use the full package name when conflict with names is occurring.

Reply:
but we could use import alias, it looks much better

`import com.google.testing.model.Locale as GoogleApiLocale`
and then
`internal fun List<GoogleApiLocale>.toApiModel(): List<Locale>`

for me looks much more better than full import

Reply:
👍